tiny_gltf.h

```diff
@@ -7579,11 +7617,40 @@ static void SerializeGltfMaterial(const Material &material, detail::json &o) {
   }
 
   SerializeParameterMap(material.additionalValues, o);
-#else
-
 #endif
 
   SerializeExtrasAndExtensions(material, o);
+
+  // MSFT_lod
+  if (!material.lods.empty()) {
```

Comment:
This is so much boiler plate, now for three extensions (see the other two above), I would need some helper functions to make this more compact.